L.Polyline.SnakeAnim.js

```diff
@@ -108,8 +110,8 @@ L.Polyline.include({
 		}
 
 		this._snakingDistance += forward;
-
-		var percent = this._snakingDistance / distance;
+		
+		let percent = distance ? this._snakingDistance / distance : 0;
```

Comment:
Please use `var` instead of `let` - Leaflet and its plugins still use ES5 syntax.